plugins/available/nvm.plugin.bash

```diff
@@ -6,7 +6,13 @@ cite about-plugin
 about-plugin 'node version manager configuration'
 
 export NVM_DIR="$HOME/.nvm"
```

Comment:
Nope, see the [recipe](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/nvm.rb#L27). They run `nvm.sh` from the brew location:

```console
○ → brew --prefix nvm
/usr/local/opt/nvm
```

But `~/.nvm` is where the node versions get installed on a per-user basis.

```console
○ → ls ~/.nvm/bin
node-v0.10.28-darwin-x64 node-v0.12.6-darwin-x64  node-v0.12.7-darwin-x64  node-v4.2.1-darwin-x64   node-v5.1.0-darwin-x64   node-v5.1.1-darwin-x64   node-v5.2.0-darwin-x64
```

Essentially, the brew install instructions don't have you copy `nvm.sh` to `~/.nvm/` so we need to run it from where it exists.